LayoutTests/imported/w3c/web-platform-tests/service-workers/service-worker/resources/fetch-request-resources-iframe.https.html

```diff
@@ -50,13 +60,42 @@
   }
 }
 
-function load_script_with_integrity(url, integrity) {
+function load_script_with_integrity(url, integrity, type) {
```

Comment:
if both script and type can be optional, maybe consider using a dictionary here:
 
```
function  load_script_with_integrity(url, { integrity, type } = { }) {
```

And then having these passed as objects.